WordPress/Classes/ViewRelated/Me/App Settings/About/AboutViewController.swift

```diff
@@ -174,33 +194,58 @@ open class AboutViewController: UITableViewController {
         UIApplication.shared.open(twitterURL)
     }
 
+    private func displayShareApp() {
+        tableView.deselectSelectedRowWithAnimation(true)
+
+        guard let sharePresenter = sharePresenter,
+              !sharePresenter.isLoading,
+              isRecommendAppSectionEnabled else {
+            return
+        }
+
+        sharePresenter.present(for: .wordpress, in: self)
```

Comment:
Note to self: If we do decide to enable this feature for Jetpack, I'll need to feed the value based on `AppConfiguration.isJetpack`.

Reply:
https://github.com/wordpress-mobile/WordPress-iOS/pull/17007#discussion_r689818965